__tests__/integration/test.bats

```diff
@@ -252,3 +257,48 @@ teardown() {
   assertFailedMkdocs build
   [[ "$output" == *"[mkdocs-monorepo] The file path /"*"/__tests__/integration/fixtures/error-include-wildcard-no-site-name/projects/project-a/mkdocs.yml does not contain a valid 'site_name' key in the YAML file. Please include it to indicate where your documentation should be moved to."* ]]
 }
+
+@test "sets edit url for included path pages" {
+  cd ${fixturesDir}/ok-include-path-edit-uri
+  assertSuccessMkdocs build
+
+  assertFileContains './site/index.html' 'href="https://github.com/backstage/mkdocs-monorepo-plugin/edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/index.md"'
+  assertFileContains './site/test/index.html' 'href="https://github.com/backstage/mkdocs-monorepo-plugin/edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/api/docs/index.md"'
+  assertFileContains './site/test/other/other/index.html' 'href="https://github.com/backstage/mkdocs-monorepo-plugin/edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/api/docs/other/other.md"'
+
+  [ "$status" -eq 0 ]
+}
+
+@test "sets edit url for included wildcard pages" {
+  cd ${fixturesDir}/ok-include-wildcard-edit-uri
+  assertSuccessMkdocs build
+
+  assertFileContains './site/index.html' 'href="https://github.com/backstage/mkdocs-monorepo-plugin/edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/docs/index.md"'
+  assertFileContains './site/test/index.html' 'href="https://github.com/backstage/mkdocs-monorepo-plugin/edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/projects/api/docs/index.md"'
+  assertFileContains './site/test/other/other/index.html' 'href="https://github.com/backstage/mkdocs-monorepo-plugin/edit/fix-edit-page-url/__tests__/integration/fixtures/bug-include-path-edit-uri/projects/api/docs/other/other.md"'
+
+  [ "$status" -eq 0 ]
+}
+
+@test "only set edit_uri for included paths if repo_url is configured" {
```

Comment:
I was thinking it'd be useful for the repo/edit URLs to be inferred for sub-urls.  Could be a little onerous to require that users define both in every `mkdocs.yml` within the monorepo.

For example: if `./mkdocs.yml` has `repo_url` and `edit_uri` configured, and also has a `!include api/mkdocs.yml` which _doesn't_ have `repo_url` and `edit_uri` configured, it seems like we still have all the information we need to build the edit URL for everything under `./api/`, right?

You could reconstruct it as `[directory containing root mkdocs.yml] minus [value of docs_dir config (defaulting to /docs)] plus [directory pointed to by !include statement] plus [value of sub-mkdocs.yml's docs_dir (defaulting to /docs)]`

Reply:
Done 🙌🏻 , could you take a look again?